website/content/en/docs/contribution-guidelines/developer-guide.md

```diff
@@ -1,97 +1,62 @@
-To build the binary and run all tests (assuming you have a correctly configured environment),
-you can simple run:
-
-```sh
-$ make test-ci
-```
-
-If you simply want to run the unit tests, you can run:
-
-```sh
-$ make test
-```
-
-For more information on running testing and correctly configuring your environment,
-refer to the [`Running the Tests Locally`][running-the-tests] document. To incorporate code changes in your development environment see the [`Testing changes Locally`][testing-changes-locally] document.
-
-To run the lint checks done in the CI locally, run:
-
-```sh
-$ make lint
-```
-
-**NOTE** Note that for it is required to install `golangci-lint` locally. For more info see its [doc](https://github.com/golangci/golangci-lint#install)
-
-## How the operator-sdk binaries are built
-
-In the release process, the script [.release.sh][release-sh] is executed and it will call the [makefile][makefile] target [make release](https://github.com/operator-framework/operator-sdk/blob/master/Makefile#L113). To know more about the release process, see the [doc][release-doc] also see [operator-sdk releases](https://github.com/operator-framework/operator-sdk/releases).
-
-**NOTE** The Deploy stage (configured in [.travis.yml ][travis]) builds also execute the same [makefile][makefile] targets. This stage is executed against the master branch when a Pull Request is merged.
-
-## How to test the build of operator-sdk binaries
-
-Follow these steps to execute the Travis `Deploy` stage against your branch to demonstrate that the merge build will complete as expected.
+See the [testing][dev-testing] and [documentation][dev-docs] guides for more information.
 
-- Enable the Travis in your fork repository. For more info see [`To get started with Travis CI using GitHub`](https://docs.travis-ci.com/user/tutorial/#to-get-started-with-travis-ci-using-github) 
-- Create image repos in quay (or another registry that supports multi-arch images) for ansible, helm, and scorecard proxy. For each image type, you need repos for the manifest list and one for each architecture (e.g. `ansible-operator`, `ansible-operator-amd64`, `ansible-operator-s390x`, etc.)
+## Releasing
 
-**NOTE** Be sure to make each repository public.
+See the [release guide][dev-release] for more information.
 
-- Set the following environment variables in the Travis settings for your fork:
+## Continuous Integration (CI)
 
-    - `ANSIBLE_IMAGE` docker image name (e.g. `quay.io/joelanford/ansible-operator`)
-    - `HELM_IMAGE` same as above, but for helm
-    - `SCORECARD_PROXY_IMAGE` same as above, but for scorecard proxy
-    - `DOCKER_USERNAME` credentials for your repo
-    - `DOCKER_PASSWORD` credentials for your repo
-    - `DOCKER_CLI_EXPERIMENTAL`  set to `enabled` 
-    - `COVERALLS_TOKEN`  token to integrate the project with `https://coveralls.io/`. So, enable your fork in `https://coveralls.io/` and generate a token to allow it. 
```

Comment:
How a collab will know how to set up the Travis in their fork if we are removing the info?

Reply:
These aren't being removed, see below.

Reply:
Sorry, I did not find where we are describing the ENV Vars that should be set in the fork travis. Could you please link it here?

Reply:
https://github.com/operator-framework/operator-sdk/pull/3879/files#diff-61d9e7ec7cfa2216804ddbbc36e883d2R48-R49